docs/adr/0031-process-evolution.md

```diff
@@ -0,0 +1,171 @@
+what was live at an arbitrary time in the past, and if we need to roll back,
+easier to get the right code live. Since they offer benefits and no real
+drawback we should continue to use tags.
+
+### Who uses the change logs and what are their expectations?
+
+We’ve maintained change logs in each package for many years and they are
+updated with each release. We release often enough that not all of the
+packages change between each release and we regularly update the changelogs
+just to say nothing has changed. This causes considerable churn in our
+deployment pipeline for no reason.
+
+We asked our stakeholders about the changelogs and the only people currently
+using them are QA. After [a
+conversation](https://groups.google.com/a/mozilla.com/g/fxa-staff/c/OuKEsHs67fk)
+we found an alternative that would allow us to stop writing our change logs to
+the repository and reduce our deployment churn.
+
+### Do we need to maintain versions in each package.json?
+
+Our build script currently increments the version of every package in our
+monorepo when we run it. But do we need those versions? [The docs
+say](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#version) “If
+you don't plan to publish your package, the name and version fields are
+optional.” We used to publish our packages to npm but we don’t do that anymore.
+SRE doesn't use the version field in any scripts. We use the version to [tell
+what is live](https://accounts.firefox.com/__version__) but the commit hash is
+also there and we could look at other options.
+
+So, no, we don't need the version in each package.json.
+
+## Decision Outcome
+
+TBD, but the authors are recommending option 3
+
+## Options
+
+### Option 1: The main branch is always ready to push live
+
+![An illustration of Option 1](images/0031-option1.png)
+
+Something like [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow).
+This is mainly used for continuous deployment which we aren’t aiming for so likely
+isn't a good fit here.
+
+#### Pros:
+
+- Forced robustness in DB/data schema changes
+- Never have to “roll back” a release? I could be wrong about this
+
+#### Cons:
+
+- Have to carefully juggle feature flags
+- L10n has no time to push unless we land the code early behind a feature flag
+- This would encourage feature branches which we’ve discouraged up until now to
+  avoid merging problems like conflicts
+- Doesn’t scale well with large teams
+
+### Option 2: A single long running release branch
+
+![An illustration of Option 2](images/0031-option2.png)
+
+#### Pros
+
+- Easy to understand
+
+#### Cons
+
+- We’d have to set up infrastructure somewhere else to push to production
+  making this less simple or effecient
+
+### Option 3: Long running stage and production branches
+
+![An illustration of Option 3](images/0031-option3.png)
+
+We make a new long-running production branch (essentially [GitLab Flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html)). From that link:
```

Comment:
I think this should say, "We make new long-running stage and production branches"?